tests/utils/common.py

```diff
@@ -526,19 +526,39 @@ def get_value(field):
 
 def get_packet_information(api, packet_header):
     """
-    Takes any packet_header for ex ethernet, ipv4, udp, tcp and returns
+    Takes any packet_header or header position
+    for ex ethernet, ipv4, udp, tcp and returns
     the packet information of that header
+
+    if string is passed the header is filtered by name
+    if int is passed header is filtered by index
     """
     trafficItem = api._ixnetwork.Traffic.TrafficItem.find()
     configElement = trafficItem.ConfigElement.find()
     pckt_info = {}
-    stack = configElement.Stack.find(StackTypeId=packet_header)
+    if isinstance(packet_header, int):
+        stack = configElement.Stack.find()[packet_header]
+    else:
+        stack = configElement.Stack.find(StackTypeId=packet_header)
     for field in stack.Field.find():
         pckt_info[field.DisplayName] = get_value(field)
     return pckt_info
 
 
 def validate_config(api, packet_header, **kwargs):
+    """
```

Comment:
We can rename it to validate_traffic_config instead of validate_config, we can discuss and do a mass update on all scripts